opentelemetry-sdk/src/testing/trace/span_exporters.rs

```diff
@@ -61,6 +60,8 @@ impl SpanExporter for TokioSpanExporter {
     fn shutdown(&mut self) {
         self.tx_shutdown.send(()).unwrap();
     }
+
+    fn set_resource(&mut self, _resource: &crate::Resource) {}
```

Comment:
please call this out in the sdk changelog.

Reply:
Added changelog, and the stress test. Seems we don't have benchmark tests of spans with resource to measure it.

Reply:
https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-sdk/benches/trace.rs#L72-L75 Won't this be enough?

Reply:
yes thanks. Have updated the benchmarks result in the description.